doc/man1/openssl-enc.pod.in

```diff
@@ -451,6 +451,13 @@ The B<-list> option was added in OpenSSL 1.1.1e.
 
 The B<-ciphers> and B<-engine> options were deprecated in OpenSSL 3.0.
 
+Since OpenSSL 3.0 using the B<-S> option makes the encrypted file to not
```

Comment:
delete 'to'

Reply:
Reworded according to @paulidale's proposal.